cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -675,6 +679,95 @@ def test_can_get_correct_usage_info_for_content_groups(self):
 
         self.assertEqual(actual, expected)
 
+    @ddt.data(ModuleStoreEnum.Type.split)
+    def test_can_get_correct_usage_info_with_orphan(self, module_store_type):
+        """
+        Test if content group json updated successfully with usage information even if there is
+        an orphan in content group.
+        """
+        with self.store.default_store(module_store_type):
+
+            split_course = CourseFactory.create()
+
+            # Create user partition
+            split_course.user_partitions = [
+                UserPartition(
+                    0, 'Name 0', 'Description 0',
+                    [Group(0, 'Group A'), Group(1, 'Group B'), Group(2, 'Group C')],
+                    scheme=None, scheme_id='cohort'
+                )]
+            split_course.save()
+
+            # Create a problem with content group
+            chapter = self.store.create_child(
+                self.user.id, split_course.location,
+                block_type='chapter',
+                block_id='chapter1',
+                fields={'display_name': 'Chapter1'}
+            )
+            sequential = self.store.create_child(
+                None, chapter.location,
+                block_type='sequential',
+                block_id='subsection1',
+                fields={'display_name': 'Subsection1'}
+            )
+            vertical = self.store.create_child(
+                None, sequential.location,
+                block_type='vertical',
+                block_id='vertical1',
+                fields={'display_name': 'Unit1'}
+            )
+            test_problem = self.store.create_child(
+                self.user.id,
+                vertical.location,
+                block_type='problem',
+                block_id='problem1',
+                fields={'display_name': 'Problem1', 'data': '<problem></problem>'}
+            )
+
+            # Group access content
+            group_access_content = {'group_access': {0: [1]}}
+
+            # Add 'test_problem' to content group 'Group B'
+            self.client.ajax_post(
```

Comment:
again, is there a helper method to do this directly instead of making an http request?